curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67ed0e7faf1f2d130a275a82.md

```diff
@@ -0,0 +1,131 @@
+---
+id: 67ed0e7faf1f2d130a275a82
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+Create a `results` state variable with an empty array (`[]`) value and its setter function, `setResults`, using the `useState` Hook.
```

Comment:
need to provide context here too. otherwise, campers won't know why they are doing this.

```suggestion
When the user types into the input field, a list of results should appear that match their query.

To handle this, create a state variable called `results` with an initial value of an empty array (`[]`). Also define its setter function, `setResults`, using the `useState` Hook.
```